main.go

```diff
@@ -63,7 +63,7 @@ func main() {
 
 func run(c *cli.Context) {
 	logrus.Info("Starting controller")
-	ctx := signals.SetupSignalHandler(context.Background())
+	ctx := signals.SetupSignalContext()
```

Comment:
Note to self (or future onlookers): This is essentially the same call that was previously there. Wrangler v0.8.8 made a breaking API change (in a patch release). This change is functionally equivalent.